config.go

```diff
@@ -641,6 +645,9 @@ func DefaultConfig() Config {
 		Sweeper: &lncfg.Sweeper{
 			BatchWindowDuration: sweep.DefaultBatchWindowDuration,
 		},
+		Htlcswitch: &lncfg.Htlcswitch{
```

Comment:
needs to be added to the `lncfg.Validate` call in `ValidateConfig`

Reply:
Woops fixed